README.rst

```diff
@@ -1,104 +1,136 @@
+
+Usage
+=====
+
+Once your instance is running, you can access the Django admin site and locate the ``EDUNEXT OPENEDX MULTITENANCY`` models.
+
+- **Microsites:** Store the microsite configuration.
+- **Routes:** Configure the URL for a tenant.
+- **Tenant configs:** Store the configuration for each tenant.
+- **Tenant organizations:** Link each organization with one or multiple tenants.
+
+Add ``EDNX_USE_SIGNAL = True`` in each microsite/tenant that wants to use the plugin. 
 
 Compatibility Notes
 --------------------
 
-+------------------+-----------------------+
-| Open edX Release | Version               |
-+==================+=======================+
-| Ironwood         | < 3.0                 |
-+------------------+-----------------------+
-| Juniper          | >= 3.0 < 4.0          |
-+------------------+-----------------------+
-| Koa              | >= 4.0 <= 5.1.3       |
-+------------------+-----------------------+
-| Lilac            | >= 4.0 < 6.2          |
-+------------------+-----------------------+
-| Maple            | >= 6.0                |
-+------------------+-----------------------+
-| Nutmeg           | >= 6.2                |
-+------------------+-----------------------+
-| Olive            | >= 8.0                |
-+------------------+-----------------------+
-| Palm             | >= v10.0.0            |
-+------------------+-----------------------+
-| Quince           | >= v11.0.0            |
-+------------------+-----------------------+
-
-**NOTE**: Since 6.2 version, eox-tenant does not support Django 2.2
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
++------------------+----------------+
+| Open edX Release | Version        |
++==================+================+
+| Ironwood         | < 3.0          |
++------------------+----------------+
+| Juniper          | >= 3.0 < 4.0   |
++------------------+----------------+
+| Koa              | >= 4.0 <= 5.1.3|
++------------------+----------------+
+| Lilac            | >= 4.0 < 6.2   |
++------------------+----------------+
+| Maple            | >= 6.0 < 11.6  |
++------------------+----------------+
+| Nutmeg           | >= 6.2         |
++------------------+----------------+
+| Olive            | >= 8.0         |
++------------------+----------------+
+| Palm             | >= v10.0.0     |
++------------------+----------------+
+| Quince           | >= v11.0.0     |
++------------------+----------------+
+
+⚠️ Since the 6.2 version, eox-tenant does not support Django 2.2
+
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied to be used for previous releases.
+
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-tenant version.
```

Comment:
What do you think about adding something like:
```suggestion
If the release you are looking for is not listed, please note:
- If the Open edX release is compatible with the current eox-tenant version (see `Compatibility Notes <https://github.com/eduNEXT/eox-tenant?tab=readme-ov-file#compatibility-notes>`_), the default configuration is sufficient.
- If incompatible, you can refer to the README from the relevant version tag for configuration details (e.g., `v6.2.0 README <https://github.com/eduNEXT/eox-tenant/blob/v6.2.0/README.rst>`_).
```
this way we can explain to the user what to do if the release is not compatible